Documentation/Books/Manual/Administration/Replication/Asynchronous/ServerLevelSetup.md

```diff
@@ -1,36 +1,37 @@
-Server-level Setup
+Server-level Replication Setup
 ==================
 
-This page describes the replication process based on a complete ArangoDB instance. That means that
-all included databases will be replicated. 
+This page describes the replication process for an **entire** ArangoDB instance. That means that
+all included databases will be replicated, the Follower will become a mirror of all data stored by
+the Leader
 
-Setting up a working master-slave replication requires two ArangoDB instances:
-* **master**: this is the instance that all data-modification operations should be directed to
-* **slave**: on this instance, we'll start a replication applier, and this will fetch data from the 
-  master database's write-ahead log and apply its operations locally
+Setting up a working classical master-slave replication requires always at least two ArangoDB instances:
+* **Leader**: this is the instance that all data-modification operations should be directed to
+* **Follower**: on this instance, we'll start a replication applier, and this will fetch data from the
+  Leader node's write-ahead log and apply its operations locally (there can be more than one Follower per Leader)
```

Comment:
Do we want to advertise this already?

Reply:
yes, this is not yet the Active-Passive Setup